packages/lwc-language-server/package.json

```diff
@@ -36,38 +35,39 @@
     "@salesforce/schema": "0.0.12",
     "@salesforce/user": "0.0.12",
     "babel-core": "7.0.0-bridge.0",
-    "camelcase": "^5.3.1",
-    "change-case": "^3.1.0",
-    "comment-parser": "^0.7.2",
+    "camelcase": "^6.0.0",
+    "change-case": "^4.1.1",
+    "comment-parser": "^0.7.6",
     "fast-glob": "^3.2.4",
-    "fs-extra": "^5.0.0",
+    "fs-extra": "^9.0.1",
     "normalize-path": "^3.0.0",
     "properties": "^1.2.1",
-    "vscode-html-languageservice": "^3.1.0-next.3",
+    "vscode-html-languageservice": "^3.1.0",
     "vscode-languageserver": "^5.2.1",
-    "vscode-nls": "^4.0.0",
+    "vscode-nls": "^4.1.2",
     "vscode-uri": "^2.1.2",
-    "xml2js": "^0.4.19"
+    "xml2js": "^0.4.23"
   },
   "devDependencies": {
-    "@types/babel-types": "^7.0.4",
-    "@types/fs-extra": "^5.0.4",
-    "@types/glob": "^5.0.35",
-    "@types/jest": "^25.1.2",
-    "@types/node": "^10.12.18",
+    "@lwc/engine": "0.34.8",
```

Comment:
Are we sure we can more `@lwc/engine` to be on `devDependencies` ?

Reply:
We are using this @lwc/engine on 2 places, one is a test called workspace-type.test.ts, and the other is a script called copy_typings.js that's used in the build script. Thats what i found.
Also, we're using this dependency in the other packages as Dev dependency. 
I think we should let this dependency on the devs

Reply:
Its strange, because this dependency is not imported anywhere for lwc-language-server... Its just used by the other ones for some test.
Maybe its imported but not used, i'll let the @lwc/engine onto the dependencies, and then make a check of unused dependencies (I saw that was one of the pending tasks to do)

Reply:
Also when removed, no error is showed on the tests

Reply:
We unfortunately can't fully rely on the tests not failing to show a problem. Let's figure out whether we need this dependency or not for a future PR.